services/web/server/src/simcore_service_webserver/director_v2_core.py

```diff
@@ -483,3 +483,68 @@ async def restart(app: web.Application, node_uuid: str) -> None:
         expected_status=web.HTTPOk,
         timeout=timeout,
     )
+
+
+@log_decorator(logger=log)
+async def requires_dynamic_sidecar(
```

Comment:
at some point we should unify how these calls among services. In this file there is as well a Client class that is used for the start/stop  entrypoint in the director-v2

Reply:
Now that I look at it it makes sense to at some point refactor it a bit. It is starting to show its age.